gh-data-summary.json

```diff
@@ -0,0 +1,9731 @@
+[
```

Comment:
Does this file need to be checked in to the repository?

Reply:
No, at least not to `main`. My plan is to check in generated files to `gh-pages` with a GH Action.